gae_serve/README.md

```diff
@@ -0,0 +1,78 @@
+
+## Introduction
+
+This is a simple sample that shows how to:
+
+- Create a GAE service that loads a pickled sci-kit learn model from GCS, and use it to serve prediction requests through Cloud Endpoints.
+
+The benefits of this configuration include:
+
+1. GAE's autoscaling and load balancing.
+1. Cloud Endpoints' monitoring and access control.
+
+
+## Requirements
+
+Python.
+
+GCP SDK.  gsutil.
+
+A GCP project that has GAE, GCS, and Cloud Endpoints enabled.
+
+
+## Setup
+
+1. `git clone https://github.com/GoogleCloudPlatform/ml-on-gcp`
+
+1. `cd ml-on-gcp`
+
+1. `pip install -r requirements.txt`
+
+1. The name `modelserve` is tentative which can be changed to something else, but you need to change all occurrences consistently.
+
+1. Update `modelserve.yaml`:  Replace `PROJECT_ID` with your project's id in this line:
+
+    `host: "modelserve.endpoints.PROJECT_ID.cloud.goog"`
+
+    Note that this file defines the API specifying the input `X` to be an array of arrays of floats, and output `y` to be an array of floats.  The model included in this sample code is a linear regression model with 2-dimensional inputs.
```

Comment:
Nit: why use uppercase 'X' and lowercase 'y'?

Reply:
@p-buse Uppercase 'X' is a common notation for the 2D array of inputs, and similarly, lowercase 'y' typically denotes a 1D array of outputs - throughout the scikit-learn documentation and all the slides in my data science class.

Reply:
TIL